src/components/ProfileCard/ProfileCard.styles.ts

```diff
@@ -0,0 +1,236 @@
+import { Card, Avatar, Typography, Box } from '@mui/material';
+import { styled } from '@mui/material/styles';
+
+import { greyScale } from '../../style-library/types/theme.helpers';
+import { SIZE_CONFIGS, VARIANT_CONFIGS } from './ProfileCard.constants';
+
+import type {
+  StyledProfileCardProps,
+  ProfileCardSize,
+  ProfileCardVariant,
+} from './ProfileCard.types';
+
+export const StyledCard = styled(Card, {
+  shouldForwardProp: (prop) => prop.toString().indexOf('$') !== 0,
+})<StyledProfileCardProps>(({ theme, $size }) => {
+  const sizeConfig = SIZE_CONFIGS[$size];
+
+  return {
+    padding: theme.spacing(sizeConfig.padding / 8),
```

Comment:
Padding is derived via `theme.spacing(sizeConfig.padding / 8)`. Dividing pixel values by 8 to compute spacing units is fragile and non-obvious. This approach relies on a specific theme spacing base (8px) and makes the constants hard to reason about.

**Sugerencia:**
```typescript
Change constants to store spacing units (numbers) instead of px, then call `theme.spacing(sizeConfig.paddingUnits)`. Example:

// constants: paddingUnits: 2
padding: theme.spacing(sizeConfig.paddingUnits),

This uses theme spacing API as intended and avoids magic arithmetic.
```